charts/ondat-operator/templates/operator.yaml

```diff
@@ -71,7 +78,7 @@ spec:
         - containerPort: 8443
           name: https
       securityContext:
-        runAsUser: 65532
+        runAsNonRoot: true
```

Comment:
no other option?

Reply:
This just reflects the upstream operator change: https://github.com/storageos/operator/blob/v2.6.0/config/manager/operator.yaml#L29

As what's wrong with runAsNonRoot? It's a security measure